jenkins/ci.opensuse.org/heat-templates/deploy-cleanvm.sh

```diff
@@ -50,7 +54,12 @@ case "$VERSION_ID" in
    ;;
 esac
 
-zypper --non-interactive install git-core ca-certificates-mozilla
+$zypper install git-core ca-certificates-mozilla
+
+# Make sure these packages are not present. If they present on the system, an
```

Comment:
This is not exactly elegant. I'd prefer setting allowVendorChange=True for cleanvm in general to circumvent this class of problem entirely. Thoughts?